Project.toml

```diff
@@ -10,6 +10,7 @@ FillArrays = "1a297f60-69ca-5386-bcde-b61e274b549b"
 IRTools = "7869d1d1-7146-5819-86e3-90919afe41df"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 MacroTools = "1914dd2f-81c6-5fcd-8719-6d5c9610ff09"
+Plots = "91a5bcdd-55d7-5caf-9e0b-520d859cae80"
```

Comment:
Please swap this out for [RecipesBase.jl](https://github.com/JuliaPlots/RecipesBase.jl) - it's a dependency-free package that provides only the stuff that you need for defining recipes, without any of the plotting baggage